changelog/1149.feature.rst

```diff
@@ -0,0 +1,4 @@
+Pytest no longer accepts prefixes of command-line arguments.
```

Comment:
I think this should be a "removal" changelog, as we are dropping a "feature" (as much as it was considered a misfeature).